### DISCUSSION TITLE
第6节课作业(9班)

### DISCUSSION THREAD

**Discussion by SaigeRing** [InternLM/Tutorial#405] [作业]
## 基础作业

- 使用 OpenCompass 评测 InternLM2-Chat-7B 模型在 C-Eval 数据集上的性能
进阶作业

- 使用 OpenCompass 评测 InternLM2-Chat-7B 模型使用 LMDeploy 0.2.0 部署后在 C-Eval 数据集上的性能

**Comment by ShelterWFF** [comment]
class6: http://t.csdnimg.cn/IDxSI

**Comment by li4692625** [comment]
https://blog.csdn.net/li4692625/article/details/135727122

**Comment by xiaomile** [comment]
[第五次作业](https://github.com/xiaomile/InternLM-homework/tree/main/%E4%BD%9C%E4%B8%9A5)

**Comment by KJ-Huang-sz** [comment]
https://blog.csdn.net/qq_40831032/article/details/135730596

**Comment by cookieswolf** [comment]
https://www.mubu.com/doc/5G1Ga4B3LLO

**Comment by AIzealotwu** [comment]
https://blog.csdn.net/m0_72806612/article/details/135765943?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135765943%22%2C%22source%22%3A%22m0_72806612%22%7D

**Comment by nini0919** [comment]
https://morning-gaura-eed.notion.site/841e5ac25d2f4b00a2c666e63b6c5f19?pvs=4

**Comment by haizeigh** [comment]
第六课作业 https://note.youdao.com/s/19vciU86
视频地址 https://www.bilibili.com/video/BV1Gg4y1U7uc/?vd_source=d6b5a09dda93b2c223da08c9b8241a1f
